src/databricks/labs/remorph/helpers/transpile_utils.py

```diff
@@ -0,0 +1,93 @@
+import json
+import logging
+import os
+
+from databricks.sdk import WorkspaceClient
```

Comment:
move this to context folder have them as part of transpile context. we will later refactor them using application context.

Reply:
cc: @bishwajit-db thoughts?

Reply:
makes sense

Reply:
Done, please check